foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/oxm/record/json/JsonStructureReader.java

```diff
@@ -306,7 +307,7 @@ private void parsePair(String name, JsonValue jsonValue) throws SAXException {
                         String prefix = parentLocalName.substring(0, nsIndex);
                         uri = namespaces.resolveNamespacePrefix(prefix);
                     }
-                    if (uri == null) {
+                    if (uri == null || uri == Constants.EMPTY_STRING) {
```

Comment:
uri cannot be null here

Reply:
Sorry it should happens due line 308 `uri = namespaces.resolveNamespacePrefix(prefix);` .

Reply:
you're right